docs/_posts/2020-02-28-puppetwash.md

```diff
@@ -0,0 +1,283 @@
+---
+title: "Puppet + Wash"
+description: "Navigate your Puppet infrastructure in Wash"
+author: michaelsmith
+---
+
+A full Puppet installation - masters, agents, database, etc - contains a lot of knowledge about your infrastructure. Sometimes this can be challenging to get at. [Puppetwash](https://github.com/puppetlabs/puppetwash) exposes that knowledge, starting with making your PuppetDB data more accessible. I'll demonstrate it by setting up an example with [Pupperware](https://github.com/puppetlabs/pupperware).
+
+## Start Pupperware
```

Comment:
I think a brief summary of the steps would be useful. It looks like what you're doing is:

* `docker-compose` to start the containers
* Check that PuppetDB's running by viewing the container's logs
* Copy the certificates over for puppetwash credentials
* Trigger a puppet run so that something can be viewed in puppetwash